internal/gensnippets/gensnippets.go

```diff
@@ -99,6 +103,50 @@ func Generate(rootDir, outDir string, apiShortnames map[string]string) error {
 	return nil
 }
 
+// GenerateSnippetsDirs takes in specified modules in rootDir and outputs their examples in outDir.
+func GenerateSnippetsDirs(rootDir, outDir string, apiShortNames map[string]string, dirs []string) error {
```

Comment:
When we copy functions like this is is a good idea to have the function with less args call the function with more args with some default value set for the new arg. This way both sets of code flow through the same logic and there is only one spot to change in the future.

Reply:
Done